src/includes/performance/add-spans-example/python.mdx

```diff
@@ -0,0 +1,34 @@
+## Add More Spans to the Transaction
+
+The next example contains the implementation of the hypothetical `process_item` function called from the code snippet in the previous section. Our SDK can determine if there is currently an open transaction and add all newly created spans as child operations to that transaction. Keep in mind that each individual span also needs to be manually finished; otherwise, spans will not show up in the transaction. When using spans and transactions as context managers, they are automatically finished at the end of the `with` block.
+
+In cases where you want to attach Spans to an already ongoing Transaction you can use `Hub.current.scope.transaction`. This property will return a `Transaction` in case there is a running Transaction otherwise it returns `None`.
+
+Alternatively, instead of adding to the top-level transaction, you can make a child span of the current span, if there is one. Use `Hub.current.scope.span` in that case.
+
+You can choose the value of `op` and `description`.
+
+```python
+import sentry_sdk
+
+def process_item(item):
+    transaction = Hub.current.scope.transaction
+    # omitted code...
+    with sentry_sdk.start_span(op="http", description="GET /") as span:
+        response = my_custom_http_library.request("GET", "/")
+        span.set_tag("http.status_code", response.status_code)
+        span.set_data("http.foobarsessionid", get_foobar_sessionid())
```

Comment:
@untitaker - can you please review?

Reply:
Most stuff in the SDK namespace has very generic naming, prime example is `init` and `Client`. Those can easily collide with user code which is why at some point I moved everything to use fully-qualified names. I don't have strong opinions on this since every developer should be able to fix this themselves and perhaps it does just make the code examples more cluttered.

Reply:
I don't have a lot of experience. If @untitaker has seen collisions then maybe it's better to change it to

```suggestion
import sentry_sdk

def process_item(item):
    transaction = sentry_sdk.Hub.current.scope.transaction
    # omitted code...
    with sentry_sdk.start_span(op="http", description="GET /") as span:
        response = my_custom_http_library.request("GET", "/")
        span.set_tag("http.status_code", response.status_code)
        span.set_data("http.foobarsessionid", get_foobar_sessionid())
```